docs/quick-start/nginx-ingress/README.md

```diff
@@ -0,0 +1,131 @@
+# Cert-Manager NGINX Ingress Quick-Start Guide
+
+## Step -1 - Install Helm Client
+**Skip this section if you have `helm` installed.**
+
+The easiest way to install `cert-manager` is to use [Helm](https://github.com/kubernetes/helm), a templating and deployment system for Kubernetes resources.
```

Comment:
This line can probably be removed. It's not actually relevant until step 3, and we should ideally link out to the official deployment docs too (as that does/should then also explain alternative deployment strategies than just Helm, e.g. the statically generated manifests we provide in `docs/deploy/`)